lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -64,8 +64,27 @@ route_distinguisher:
   _exclude: [ios_xr]
   get_value: '/^rd (\S+)$/'
   set_value: '<state> rd <rd>'
+  context: ['vrf context <vrf>']
```

Comment:
I guess, yeah - it would be better to make that refactoring change in develop and inherit it up to feature/grpc.